README.md

```diff
@@ -147,26 +147,28 @@ Once you've done that, restart the IDE.
 
 ## Run Locally
 
-* Build and run tests
```

Comment:
Please keep this logic as it is and another step to create jar, because it is important to know how to build and run tests.